src/Services/AppInfoService.cs

```diff
@@ -2,22 +2,33 @@
 // Licensed under the MIT license.
 
 using System.Reflection;
+using System.Security.Principal;
 using DevHome.Common.Helpers;
 using DevHome.Common.Services;
 using DevHome.Helpers;
 using Windows.ApplicationModel;
 
 namespace DevHome.Services;
 public class AppInfoService : IAppInfoService
-{
+{
+    private static bool RunningAsAdmin
+    {
+        get
+        {
+            using var identity = WindowsIdentity.GetCurrent();

```

Comment:
Would a static constructed be better for computing `IsInRole`?  With a static constructor `IsInRole` is computed only once and then runningAsAdmin does not need to compute `IsInRole` on every `get` call.

Reply:
Possibly, but it's already only called at app startup.  If it's needed elsewhere, we can move it then.